SYCL/ESIMD/api/functional/type_coverage.hpp

```diff
@@ -0,0 +1,106 @@
+//===-- logger.hpp - Declarate functions and containers with that will be used
```

Comment:
Please copy-paste error here, this line has wrong file name and probably description as well.

Reply:
The description of `type_coverage.hpp`-file was changed [here](https://github.com/intel/llvm-test-suite/pull/565/commits/d56ee8f4b7fd7c3b8f8d3015a45caef1e6bda23e#diff-3bbc1ab1445c827282cc08d73f76b58cf553d9ced2c799ba46e6148583896d22L1-R10)

Reply:
Thank you. I am marking this as resolved now.